packages/ui-dom-utils/src/index.ts

```diff
@@ -50,6 +50,5 @@ export { ownerDocument } from './ownerDocument'
 export { ownerWindow } from './ownerWindow'
 export { requestAnimationFrame } from './requestAnimationFrame'
 export type { RequestAnimationFrameType } from './requestAnimationFrame'
-export { supportsObjectFit } from './supportsObjectFit'
```

Comment:
Hm, removing it now like this is a breaking change, someone might use this util somewhere. Should we mark it deprecated for now and remove it in v9?

Reply:
Since it was a util needed for IE11, and we dropped support for it a long time ago, this doesn't count as breaking change.